features/support/steps.js

```diff
@@ -103,15 +121,11 @@ Then('the report will be saved in a file called {string}', function (fileName) {
 });
 
 Then(/^the report name on the sidebar (?:is|will be) '(.+)'$/, function (reportTitle) {
-  return this.app.client.getText('#sidenavTitle').then((reportText) => {
-    expect(reportText).to.eql(reportTitle);
-  });
+  return this.app.client.$('#sidenavTitle').then((sideNavElem) => sideNavElem.getText()).then((text) => expect(text).to.equal(reportTitle));
```

Comment:
could you wrap these a bit so it's cleaner?

```suggestion
  return this.app.client.$('#sidenavTitle')
    .then((sideNavElem) => sideNavElem.getText())
    .then((text) => expect(text).to.equal(reportTitle));
```